js/templates/modals/about/btcticker.html

```diff
@@ -0,0 +1,5 @@
+<div class="pureFlex cent currentBTCPrice clrP clrBr clrSh3 tx4 contentBox padSm">
```

Comment:
Why does this have new classes instead of using flexVCent? As far as I can tell, the existing flexVCent would result in the same layout.